### PR TITLE
Added new content and media.

### PR DESCRIPTION
**Pull request by jessyrod68**
## OS Variable Refresh Rate (Build 18312)
With Windows Version 1903, we have added a new toggle in Graphics Settings for variable refresh rate. Variable refresh rate (VRR) is similar to NVIDIA’s G-SYNC and VESA DisplayPort Adaptive-Sync. For more details, read the blog post [here](https://devblogs.microsoft.com/directx/os-variable-refresh-rate/)
OS Variable Refresh Rate

I don't know if I did the add media part right, can someone please assist.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: wip/at-home/Whats-new-wip-at-home-19h1.md [hunk 1]
```diff
@@ -136,6 +136,10 @@ We are introducing a simplified Windows Insider Program Settings page via Settin
 
 ![Windows Insider Program Settings](images/18317-3.jpg "A simpler Windows Insider Program Settings page")
 
+## OS Variable Refresh Rate (Build 18312)
+With Windows Version 1903, we have added a new toggle in Graphics Settings for variable refresh rate. Variable refresh rate (VRR) is similar to NVIDIA’s G-SYNC and VESA DisplayPort Adaptive-Sync. For more details, read the blog post [here](https://devblogs.microsoft.com/directx/os-variable-refresh-rate/)
```
Comment (JohanFreelancer9) [inline wip/at-home/Whats-new-wip-at-home-19h1.md]:
```suggestion
With Windows Version 1903, we have added a new toggle in Graphics Settings for Variable Refresh Rate  (VRR). VRR is similar to NVIDIA’s G-SYNC and VESA DisplayPort Adaptive-Sync. For more details, read the [blog post](https://devblogs.microsoft.com/directx/os-variable-refresh-rate/).
```

**Comment by mypil** [comment]
@jessyrod68 - we need your kind assistance to merge the new commit for the copy/edit review made by JohanFreelancer9 / nenonix or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by jessyrod68** [comment]
Done let me know if you need anything esle from me. 

Thanks

**Comment by jessyrod68** [comment]
Done let me know if you need anything esle from me.
Thanks
@mypil

**Comment by mypil** [comment]
@lizap - Can you please assist to check, approve and merge the content of this PR?

Thank you.

**Comment by mypil** [comment]
@lizap - We would like to follow up. Can you please assist to check, approve and merge the content of this PR?

Thank you.

**Comment by mypil** [comment]
@lizap - Can you please assist to check, approve and merge the content of this PR?

Thank you.

cc: @dougkim

**Comment by mypil** [comment]
@lizap - We would like to follow up. PR is ready for final review. Thank you.

**Pull request merged**